datalink/mstp.go

```diff
@@ -0,0 +1,95 @@
+// +build MSTP
```

Comment:
Looks like a WIP. Before removing this tag, I would like to remove the cgo dependency to keep this library pure go. For now this is fine though since I imagine that is a bit of work.

Reply:
If this is a WIP that you want useable, could you add this to the readme as well.

Reply:
What do you mean by WIP？
MS/TP is another datalink based on serial port 485 and its implemention seems very complex, so I use a C library directly for a temporary solution. To be implemented by pure go.

Reply:
you can ignore this file.
If not set -tags MSTP, this file won't be compiled